setup.py

```diff
@@ -11,13 +12,46 @@
 from rpm_py_installer.version import VERSION
 
 
+def run_cmd(cmd):
+    """Run a command."""
+    exit_status = 0
+    message = ''
+    message_format = 'Command: {0}'
+
+    # The subprocess.run() function was added in Python 3.5.
+    # https://docs.python.org/3.5/library/subprocess.html
+    if sys.version_info > (3, 5):
+        p = subprocess.run(
+            cmd.split(' '), stdout=subprocess.PIPE, stderr=subprocess.PIPE
+        )
```

Comment:
You can use `Popen` directly (should work in any Python version):

```python
proc = subprocess.Popen(cmd.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
proc.wait()
exit_status = proc.returncode
stdout = proc.stdout.read().decode()
stderr = proc.stderr.read().decode()
```

Reply:
Thanks. That makes sense. I will fix on your way. I was afraid that the way with `Popen` made the code complicated. But it seems that it is actually still simple.